src/EVM/Expr.hs

```diff
@@ -785,6 +792,9 @@ simplify e = if (mapExpr go e == e)
     -- Double NOT is a no-op, since it's a bitwise inversion
     go (EVM.Types.Not (EVM.Types.Not a)) = a
 
+    go (Max (Lit 0) a) = a
+    go (Max a (Lit 0)) = a
```

Comment:
I think we don't need this match anymore since you used `normArgs` for `min` and `max` below?

Reply:
Done! Also added the corresponding case for Min